products/oneplus.md

```diff
@@ -0,0 +1,103 @@
+    label: Supported Android
+    description: Supported Android versions range
+    link: https://endoflife.date/android
+
+###### Supported Android versions ######
+# Because OnePlus is using community forum posts as official releases posts it is hard to parse the last Android/OxygenOS releases 
+# for all devices so I'm using the unofficial https://oxygenupdater.com/news/all/ that have been collecting all OnePlus OxygenOS announcements for several years and is an established and trusted way to get OP updates.
+# still not great but easier to search and parse than OnePlus Community and more up to date than GsmArena
+# There is also https://www.androidupdatetracker.com/policy/oneplus that is easier to parse and up to date but AFAIK is less established
+
+###### List of Phones ######
+# for a list of released phones https://www.gsmarena.com/oneplus-phones-95.php
+
+releases:
+-   releaseCycle: "12"
+    releaseLabel: "OnePlus 12"
+    releaseDate: 2023-12-05
+    support: 2027-12-05 # approximation "4 major Android updates"
+    eol: 2028-12-05
+    discontinued: false
+    link: https://www.gsmarena.com/oneplus_12-12725.php
+    supportedAndroidVersions: 14 # https://oxygenupdater.com/article/396/
+
+-   releaseCycle: "11r"
+    releaseLabel: "OnePlus 11R"
+    releaseDate: 2023-02-07
+    support: 2027-02-07 # approximation "4 major Android updates"
+    eol: 2028-02-07
+    discontinued: false
+    link: https://www.gsmarena.com/oneplus_11r-11915.php
+    supportedAndroidVersions: 13 - 14 # https://oxygenupdater.com/article/431/
+
+-   releaseCycle: "11"
+    releaseLabel: "OnePlus 11"
+    releaseDate: 2023-02-07
+    support: 2027-02-07 # approximation "4 major Android updates"
+    eol: 2028-02-07
+    discontinued: false
+    link: https://service.oneplus.com/us/search/search-detail?id=2123192&articleIndex=2
+    supportedAndroidVersions: 13 - 14 # https://oxygenupdater.com/article/426/
+
+
+# https://community.oneplus.com/thread/1462181
+# starting with the 8, the release policy was 3 major android updates and 4 years of security updates (    
+#    OnePlus 10 Pro
+#    OnePlus 10R
+#    OnePlus 10T
+#    OnePlus Ace
+#    OnePlus Ace Racing Edition
+#    OnePlus 9RT
+#    OnePlus 9 Pro
+#    OnePlus 9R
+#    OnePlus 9
+#    OnePlus 8T
+#    OnePlus 8 Pro
+#    OnePlus 8)
+#    OnePlus Nord 3 https://community.oneplus.com/thread/1356800969827942405
+# starting with the 8, the release policy was 2 major android updates and 3 years of security updates (    
+#    OnePlus 7 Pro
+#    OnePlus 7T Pro
+#    OnePlus 7T
+#    OnePlus 7
+#    OnePlus Nord CE
+#    OnePlus Nord
+#    OnePlus Nord 2
+#    OnePlus Nord CE 2 Lite))
+# these devices will get 1 major android update and 3 years of security updates
+#    OnePlus N200 5G
+#    OnePlus N100
+#    OnePlus N10 5G
+
+
+
+---
+
+> OnePlus is a Chinese manufacturer of Android phones and other consumer electronics.
+
+OnePlus phones receives updates every two months.
```

Comment:
Are all OnePlus phones running OxygenOS ? If so I think it would be useful to add a paragraph at the start of the description about the relation between OxygenOS and Android, and then never mention Android again.

Reply:
I've added a small sentence to explain. Regarding mentioning Android or OxygenOS throughout the page, see my answer on the custom column conversation